### PR TITLE
Support Jaeger storage flags as default options

### PR DESCRIPTION
**Pull request by pavolloffay**
Related to #7 

This PR adds Jaeger ES storage flags to the OTEL command. Flag values are used as default values for Elasticsearch configuration.

Viper is also configured so the flags can be passed as env variables.

Signed-off-by: Pavol Loffay <ploffay@redhat.com>

### CODE REVIEW

File: pkg/exporter/elasticsearch/factory.go [hunk 1]
```diff
@@ -12,19 +12,19 @@ import (
 
 const (
 	typeStr = "jaeger_elasticsearch"
```
Comment (objectiser) [inline pkg/exporter/elasticsearch/factory.go]: Although not part of this PR, just wondering whether (as this exporter is located in the Jaeger repo) whether we need the `jaeger_` prefix?

And at some point, when this component is perhaps not so Jaeger specific - and operating off the OpenTelemetry proto model, it might be a candidate for moving to the contrib repo.

Comment (pavolloffay) [reply pkg/exporter/elasticsearch/factory.go]: That would make sense if the module was storing OTEL model and there was also supported OT query API.

Comment (pavolloffay) [reply pkg/exporter/elasticsearch/factory.go]: On the `jaeger` prefix: 

I think it makes sense to distinguish Jaeger components. At the moment there is `jaeger` receiver, `jaeger_grpc` and `jaeger_thrift_http` exporters. For the consistency reasons, I would keep the prefix. If there will be plans in the future to support storage layer in OTEL community those probably will not have vendor prefix.

**Comment by pavolloffay** [comment]
@objectiser PR updated and ready for re-review

File: cmd/collector/main.go [hunk 2]
```diff
@@ -24,15 +28,24 @@ func main() {
 		//GitHash:  version.GitHash,
 	}
 
+	// TODO should we support jaeger conf file?
```
[the remark this answers is not in the document]
Comment (pavolloffay) [reply cmd/collector/main.go]: This is tracked in https://github.com/jaegertracing/jaeger-opentelemetry-collector/issues/7#issuecomment-592601132. The proper viper configuration with watching the config file should solve this for all viper managed properties.

**Comment by pavolloffay** [comment]
> Apart from a couple of comments, and changes, looks good to me - but may be better to get someone with more viper knowledge to confirm.

@jpkrohling would you like to have a look?
